pkg/toolchains/rpmbuild_configure.bzl

```diff
@@ -0,0 +1,53 @@
+# Copyright 2020 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Repository rule to autoconfigure a toolchain using the system rpmbuild."""
+
+def _write_build(rctx, path):
+    if not path:
+        path = ""
+    rctx.template(
+        "BUILD",
+        Label("//toolchains:BUILD.tmpl"),
+        substitutions = {
+            "{GENERATOR}": "@rules_pkg//toolchains/rpmbuild_configure.bzl%find_system_rpmbuild",
+            "{RPMBUILD_PATH}": str(path),
+        },
+        executable = False,
+    )
+
+def _find_system_rpmbuild_impl(rctx):
+    rpmbuild_path = rctx.which("rpmbuild")
+    if rctx.attr.verbose:
+        if rpmbuild_path:
+            print("Found rpmbuild at '%s'" % rpmbuild_path)
+        else:
+            print("No system rpmbuild found.")
```

Comment:
It would be nice if this explanation could be propagated to `pkg_rpm` somehow.  As it stands, when `rpmbuild` absent but needed, it is possible for a significant amount of text to be between this message and the actual output, potentially confusing users.

I feel that this could be deferred.

Reply:
An alternative is to fail hard if you ask for it and don't find it. I can't do that here because I want these rules to run on all platforms.

Let's defer this. There are a lot of UX issues. Some people will want to fail at WORKSPACE load time and others will want the individual targets to fail and/or skip if they try to build rpms from Windows.

Reply:
Deferred to #268